contracts/root/Vesting.sol

```diff
@@ -25,13 +27,18 @@ contract Vesting is Ownable {
     mapping(address => uint256) public userPlanId;
     mapping(address => uint256) public allocations;
     mapping(address => uint256) public claimed;
+    mapping(address => uint256) public vtSQTAllocations;
 
     event VestingPlanAdded(uint256 planId, uint256 lockPeriod, uint256 vestingPeriod, uint256 initialUnlockPercent);
     event VestingAllocated(address indexed user, uint256 planId, uint256 allocation);
     event VestingClaimed(address indexed user, uint256 amount);
+    event TokenDeposited(address indexed user, uint256 amount);
```

Comment:
We can use vtSQT's transfer event to track.

Reply:
It may easier for subquery project to handle it within the same contract, otherwise need to extra logic to check the from and to address to identity deposit and withdraw action